.gitattributes

```diff
@@ -0,0 +1,2 @@
+# Do not convert line endings as is directly compared with in spec
+spec/fixtures/saved-html.html -text
```

Comment:
I could go either way. This tells it to leave the file as-is - the other pattern forces specific line endings.